Tests/PackageGraphPerformanceTests/DependencyResolverPerfTests.swift

```diff
@@ -174,14 +174,17 @@ extension ProductFilter: JSONSerializable, JSONMappable {
         switch self {
         case .everything:
             return "all".toJSON()
-        case .specific(let products):
+        case .specific(var products, let includeCommands):
+            if includeCommands { products.insert("_commands_") }
```

Comment:
I know it's a nitpick, but perhaps call this "command plugins" since command is a somewhat overloaded term?

Reply:
Which are you referring to? The label of the enumeration’s associated value, or the JSON placeholder string?

Reply:
Both actually, but the name of the placeholder is one that might show up in debug output (I assume?), so that's the more important one, I'd assume.

Reply:
@swift-ci please smoke test

Reply:
I did find‐and‐replace from “includeCommands” to “includeCommandPlugins” and from “_commands_” to “_command_plugins_”.

Reply:
Thank you!  Might save someone some confusion one day...